packages/components/src/core/Dropdown/index.tsx

```diff
@@ -1,24 +1,30 @@
+import { AutocompleteProps } from "@mui/material";
 import {
   AutocompleteCloseReason,
   AutocompleteValue,
 } from "@mui/material/useAutocomplete";
-import React, { ReactNode, useEffect, useState } from "react";
+import React, { ReactNode, useState } from "react";
+import {
+  AutocompleteMultiColumnOption,
+  AutocompleteSingleColumnOption,
+  DefaultAutocompleteOption,
+} from "../Autocomplete";
 import DropdownMenu, {
-  DefaultDropdownMenuOption,
   DropdownMenuProps as SdsDropdownMenuProps,
 } from "../DropdownMenu";
 import { StyledPaper, StyledPopper } from "../DropdownMenu/style";
 import InputDropdown, {
   InputDropdownProps as InputDropdownPropsType,
 } from "../InputDropdown";
-import { StyledButton } from "./style";
+import { StyledButton, StyledButtonsWrapper } from "./style";
 
 export {
   InputDropdown as DropdownInputDropdown,
   StyledPaper as DropdownPaper,
   StyledPopper as DropdownPopper,
 };
 
+// (masoudmanson): This can be removed since we no longer need it
```

Comment:
should we remove this type in this PR or later? Thanks!

Reply:
Of course, I just removed them. 🫠